Config options for rescaling media costs (#1041)
gamma-delta/HexMod: CHANGELOG.md

### Added

- Added the `cannot_modify_cost` tag for patterns that should ignore the `media_consumption` attribute when calculating cost, by Robotgiggle in [987](https://github.com/FallingColors/HexMod/pull/987).

> - Added a config options to rescale the cost of some or all patterns ([#1041](https://github.com/FallingColors/HexMod/pull/1041)) @Robotgiggle

Reviewer: Small typo: "a config options"

Nit: Wording could be improved for clarity.
```suggestion
- Added config options to scale pattern costs per action and globally ([#1041](https://github.com/FallingColors/HexMod/pull/1041)) @Robotgiggle
```
